backend/LeetWars.Core/LeetWars.Core.BLL/Services/UserService.cs

```diff
@@ -88,9 +90,9 @@ public async Task<bool> CheckIsExistingEmailAsync(string email)
 
     public async Task<bool> CheckIsExistingUserNameAsync(string? userName)
     {
-        if (string.IsNullOrEmpty((userName)))
+        if (string.IsNullOrEmpty(userName))
         {
-            return false;
+            return true;
         }
 

```

Comment:
why true here?

Reply:
to generate unique usernames for github/google if no username provided